content/06-lf.md

```diff
@@ -278,6 +278,22 @@ In `Main.hx` we can use `somepackage.Example` module. This module is defined in
 
 
 
+<!--label:lf-custom-defines-documentation-->
+#### Custom defines documentation
+
+##### since Haxe 4.3.0
+
+As noted in [conditional compilation](lf-condition-compilation), custom defines can always be added on the command line with `-D ...`. However, to provide better completion in the IDE, and to provide documentation for a Haxe user, defines can be registered with the compiler in [initialization macros](macro-initialization) using the methods:
+
+* [`registerDefinesDescriptionFile`](https://api.haxe.org/v/development/haxe/macro/Compiler.html#registerDefinesDescriptionFile)
```

Comment:
Hmm we still need to deploy API docs for 4.3.0... and then use 4.3 doc links here I suppose.